### DISCUSSION TITLE
Ideas for the Events Page

### DISCUSSION THREAD

**Discussion by mghabs** [memser-spaceport/pln-events#96] [Ideas]
Amazing effort, thank you so much for getting this live!

Brain-dumping some ideas here:

1. Would be great to see a "save to calendar" option which is easy to access on the homepage rather than going to each event page, where some don't have that option.
2. Even simpler would be a calendar subscription which auto-pushes new events to subscribers' calendars
3. Per event it would be great to see headline facts such as:
- prices starting from
- PLN discounts available 
- contacting the organisers or PL PoC

**Comment by pandeyanujk** [comment]
Thanks for the feedback @mghabs we will take these ideas into consideration as we plan future iterations. A mobile-friendly version is dropping soon! Cc @abarna-visvanathan
